// Copyright 2024 Ant Group Co., Ltd.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

option cc_generic_services = true;

service SfFedProxy {
  rpc SendData(SfFedProxySendData) returns (SfFedProxySendDataResponse) {}
}

message SfFedProxySendData {
  bytes data = 1;
  int64 seq_id = 2;
  string job_name = 3;
};

message SfFedProxySendDataResponse {
  int32 code = 1;
  string result = 2;
};
